Formula/harmony-one.rb

```diff
@@ -0,0 +1,113 @@
+class HarmonyOne < Formula
+  desc "Harmony Blockchain Node"
+  homepage "https://github.com/harmony-one/harmony"
+  # CONTRIBUTORS: Update the following (url, sha256)
+  url "https://github.com/harmony-one/harmony/archive/refs/tags/v4.3.0.tar.gz"
+  sha256 "d463cd60973509f2a7e906582a09a7cd21cf3638b956f164c63e0536a2063367"
+  license "LGPL-3.0-or-later"
+
+  # CONTRIBUTORS: Versions may need to be updated according to documentation at:
+  #    -  https://github.com/harmony-one/harmony
+
+  depends_on "bash"
+  depends_on "gmp"
+  depends_on "go@1.16"
```

Comment:
Why not a newer Go?

Reply:
because thats what the build specs in the readme.md state as a dependency?

https://github.com/harmony-one/harmony

Reply:
Homebrew always uses the latest version, if that's not possible we need to know why